docs/core-services/nvidia-triton-server-inference-engine.md

```diff
@@ -263,4 +180,105 @@ then encrypt it with the AES 256 algorithm using the following `gpg` command:
 gpg --armor --symmetric --cipher-algo AES256 tf_autoencoder_fp32.zip
 ```
 
-The resulting archive `tf_autoencoder_fp32.zip.asc` can be transferred to the _Local Model Repository Path_ on the target machine and will be decrypted by Kura.
\ No newline at end of file
+The resulting archive `tf_autoencoder_fp32.zip.asc` can be transferred to the _Local Model Repository Path_ on the target machine and will be decrypted by Kura.
+
+## Triton Server Metrics and Statistics support
+
+Since version 6.0.0, Eclipse Kura supports metrics and statistics reporting from a generic Inference Engine, leveraging the [InferenceEngineMetricsService](https://github.com/eclipse-kura/kura/blob/develop/kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/ai/inference/InferenceEngineMetricsService.java) APIs.
+
+The implementation for the Triton Server allows to retrieve relevant metrics regarding GPU and models from the engine. It is based on the [Nvidia Triton Server Metrics feature](https://github.com/triton-inference-server/server/blob/r24.08/docs/user_guide/metrics.md) and the [Model Statistics Extension](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_statistics.md). The feature is enabled using the `Enable Triton Server Metrics` parameter from the Eclipse Kura webUI or the `enable.metrics` property in the snapshot configuration. This property is available only for the Triton Server Native Service and Triton Server Container Service components. In the case of the Triton Server Remote Service, the metrics reporting cannot be configured but it can be available depending on the remote Triton Server setup.
```

Comment:
Shouldn't we update all the above descriptor so that they list the `Enable Triton Server Metrics` parameter in their parameters? Just to keep things consistent. It might be a simple work and then a link to this new section